src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/AddRewardTierButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { Trans } from '@lingui/macro'
+import { Button } from 'antd'
+
+import { PlusCircleOutlined } from '@ant-design/icons'
+import { CSSProperties } from 'react'
+
+export function AddRewardTierButton({
+  onClick,
+  disabled,
+  style,
+}: {
+  onClick: VoidFunction
+  disabled?: boolean
+  style?: CSSProperties
+}) {
+  const buttonStyle: CSSProperties = {
+    ...style,
+    marginTop: 15,
+    height: 60,
```

Comment:
**suggestion**: I think leave this as-is. Give the button`size="large"` and call it a day. Otherwise we're being inconsistent with other buttons